std/experimental/scripting.d

```diff
@@ -0,0 +1,83 @@
+/++
+Phobos is the D standard library, built on top of druntime.
```

Comment:
Seems obvious.

Reply:
I originally wanted that on the `package.d` to give a high level overview introduction for generated docs. It doesn't make much sense in a `scripting.d` file though...